### PR TITLE
Legacy analytics events

### PR DESCRIPTION
**Pull request by M-OC**

### CODE REVIEW

File: src/lib/legacy-analytics/analytics-init.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+import { legacyFpti } from './legacy-fpti';
+
+export const analyticsInit = (config : Config) => {
+  legacyFpti(config, {
+    fltp: 'store-cash',
+    es: 'connectionStarted',
+    website: 'muse',
+    feature: 'offer',
+    subfeature1: 'store-cash',
+    subfeature2: 'sdk',
```
Comment (joshbeam) [inline src/lib/legacy-analytics/analytics-init.js]: did you check with matt/gds on this filter?

File: src/lib/legacy-analytics/merchant-user-event.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
+/* @flow */
+import type { Config } from '../../types';
+
+import { legacyFpti } from './legacy-fpti';
+
+export const merchantUserEvent = (config : Config) => {
```
Comment (joshbeam) [inline src/lib/legacy-analytics/merchant-user-event.js]: [nit] name `merchantRecognizedUser`

File: src/lib/legacy-analytics/legacy-fpti.js [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+
+  // Source identifier ("component" in fpti team's terms)
+  sub_comp: data.subcomponent,
+
+  // Page domain, path & querystring
+  completeurl: data.completeUrl
+});
+
+export const legacyFpti = (config : Config, data : any) => {
+  const fptiServer = 'https://t.paypal.com/ts';
```
Comment (joshbeam) [inline src/lib/legacy-analytics/legacy-fpti.js]: can this be configurable for testing in different envs?

Comment (M-OC) [reply src/lib/legacy-analytics/legacy-fpti.js]: Yes. Will do a follow-up pr.

**Pull request merged**